backend/api/Services/RobotService.cs

```diff
@@ -173,6 +173,12 @@ private IQueryable<Robot> GetRobotsWithSubModels()
                 .Include(r => r.VideoStreams)
                 .Include(r => r.Model)
                 .Include(r => r.CurrentArea)
+                .ThenInclude(r => r != null ? r.Deck : null)
+                .Include(r => r.CurrentArea)
```

Comment:
is this supposed to be r.Deck ? :) Same for the other lines below

Reply:
Changed name to area